pybamm/parameters/print_parameters.py

```diff
@@ -0,0 +1,109 @@
+#
+# Print parameters
+#
+import pybamm
+from collections import defaultdict
+
+ignore = [
+    "__name__",
+    "__doc__",
+    "__package__",
+    "__loader__",
+    "__spec__",
+    "__file__",
+    "__cached__",
+    "__builtins__",
+    "absolute_import",
+    "division",
+    "print_function",
+    "unicode_literals",
+    "pybamm",
+    "constants",
+]
+
+
+def print_parameters(parameters, parameter_values, output_file=None):
+    """
+    Evaluate and print parameters to an output file.
+    For dimensionless parameters that depend on the C-rate, the value is given as a
+    function of the C-rate (either x * Crate or x / Crate depending on the dependence)
```

Comment:
is `Crate` or `C-rate` a common battery term? Perhaps we should give the full name at least once?

Reply:
`C-rate` is the full name, it is very common so all the battery people should know what this means (A C-rate of 1 C is the current required to fully discharge the battery in 1 hour, 2 C is current to discharge to the battery in 0.5 hours, etc)